Makefile

```diff
@@ -175,6 +175,14 @@ githooks:
 clippy:
 	SKIP_WASM_BUILD=1 cargo clippy --workspace --all-targets --all-features -- -D warnings
 
+.PHONY: clippyfix ## cargo clippy --fix
+clippyfix:
+	SKIP_WASM_BUILD=1 cargo clippy --fix --workspace --all-targets --all-features -- -D warnings
+
+.PHONY: cargofix ## cargo clippy --fix
```

Comment:
from the `make help` menu, both have the same comment. a bit confusing. But not a big issue.

Reply:
> from the `make help` menu, both have the same comment. a bit confusing. But not a big issue.

Thanks, will correct it in next PR